Makefile

```diff
@@ -1,10 +1,24 @@
+UNIT_TAGS :=  "$(subst :, or ,$(shell awk '{print $2}' src/test/unit.tags | paste -s -d: -))"
+INTEGRATION_TAGS := "$(subst :, or ,$(shell awk '{print $2}' src/test/integration.tags | paste -s -d: -))"
```

Comment:
cucumber tags now reside in:
* `src/test/integration.tags`
* `src/test/unit.tags`